app/controllers/api/tasks_controller.rb

```diff
@@ -16,9 +16,14 @@ def update
     if @task.users.first.id != params[:user_id]
       self.unassign(@task.users.first.id, @task.id)
       self.assign(params[:user_id], @task.id)
+    end
+    user = User.find(params[:current_user])
+    if @task.completed_status == 0
```

Comment:
Is this defined as an enum in your model? If it is then it'd be good to define it with a description so that it's more clear what 0 means

Reply:
see here http://api.rubyonrails.org/classes/ActiveRecord/Enum.html